ci/ray_ci/doc/api.py

```diff
@@ -61,9 +61,8 @@ def from_autosummary(doc: str, current_module: Optional[str] = None) -> List["AP
                 # end of autosummary, \s means empty space, this line is checking if
                 # the line is not empty and not starting with empty space
                 break
-            api_name = (
-                f"{current_module}.{line.strip()}" if current_module else line.strip()
-            )
+            attribute = line.strip().removeprefix("~")
```

Comment:
could you add a comment that explains what `~` means? similar to the one in PR description?